R/model-crate.R

```diff
@@ -1,46 +0,0 @@
-#' @rdname save_model
-#' @export
-save_model.crate <- function(model, path, model_spec = list(), ...) {
```

Comment:
i do like being able to get rid of this, in favor of using `FUN` directly where appropriate

Reply:
Yeah, I really don't get the point of this at all. It seems like they're just extending the python functionality, but if you're going to load a literal R function, why not just load the model?